### DISCUSSION TITLE
Can TTS run with Python 3.5?

### DISCUSSION THREAD

**Discussion by jeremiahrose** [coqui-ai/TTS#765] [General Q&A] [answered]
I'm asking because I'm working with an embedded device whose SDK only supports Python 3.5.

Is it possible to install / run Coqui TTS with this version of Python?

**Comment by domcross** [comment]
See section [Install TTS](https://github.com/coqui-ai/TTS#install-tts): 

> TTS has been tested with Python >=3.6 , <3.9

**Reply by jeremiahrose** [reply]
Yes. This is why I'm asking here. Just wondering if anyone has tried or had any success?

**Comment by jeremiahrose** [comment, marked as answer]
For the record, I spent some time trying to get this working, including modifying setup.py to disable the version check and downgrading dependencies one-by-one to older versions compatible with Python 3.5.
Ultimately it failed because the dependency gruut has no older versions that work with Python 3.5.
So in conclusion - not worth the effort.

**Reply by erogol** [reply]
why p3.5 ?
